### ISSUE TITLE
Question about the New loss function on Recognition (like ViT model)

### ISSUE THREAD

**Issue by andrewwang0612** [taoyang1122/adapt-image-models#26]
Hello，Thanks for the amazing project!

Hello, I would like to ask how to configure the loss function in the training of a recognition model such as ViT. Specifically, I have seen that for tasks like object detection, the loss function is set as "loss_bbox." However, I am unsure of how to set the loss configuration for recognition tasks.

Thanks for helping!

**Comment by taoyang1122** [comment]
Hi, the loss function really depends on you task. For example, in classification, people usually use cross_entroy loss. And as you mentioned, in object detection, they have both classification and regression losses. You could check different works and follow their settings.

**Comment by Fanye12** [comment]
thank u for your reply
